From 0c4083dd8fe3cc717a75116f1b20805d994cd9d0 Mon Sep 17 00:00:00 2001
From: Devaraj Ranganna <devaraj.ranganna@arm.com>
Date: Mon, 1 Apr 2019 12:49:52 +0100
Subject: [PATCH 1/1] Set variable
 EVENT__HAVE_WAITPID_WITH_WNOWAIT_EXITCODE__TRYRUN_OUTPUT to empty string to
 avoid cmake configure error.

Signed-off-by: Devaraj Ranganna <devaraj.ranganna@arm.com>
---
 lib/libevent/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/libevent/CMakeLists.txt b/lib/libevent/CMakeLists.txt
index 32c6bc6..6d7f5fa 100644
--- a/lib/libevent/CMakeLists.txt
+++ b/lib/libevent/CMakeLists.txt
@@ -2,7 +2,7 @@
 # Applies for CMake build for libevent 2.1.8-stable
 # Fixing commit: https://github.com/libevent/libevent/commit/66a4eb0c3ae3b1f22b084b2d3aeb5c872f37efbd
 set(EVENT__HAVE_WAITPID_WITH_WNOWAIT_EXITCODE 1)
-
+set(EVENT__HAVE_WAITPID_WITH_WNOWAIT_EXITCODE__TRYRUN_OUTPUT "")
 set (EVENT__DISABLE_DEBUG_MODE ON CACHE BOOL "Disable libevent debug mode.")
 set (EVENT__DISABLE_TESTS ON CACHE BOOL "Disable libevent tests.")
 set (EVENT__DISABLE_REGRESS ON CACHE BOOL "Disable libevent regression tests.")
-- 
2.20.1

